src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_FOR_RENT = 40;
+
+  const AMOUN_DAY_FOR_MIN_DISCOUNT = 3;
+  const AMOUN_DAY_FOR_MAX_DISCOUNT = 7;
+
+  const MIN_DISCOUNT = 20;
+  const MAX_DISCOUNT = 50;
+
+  if (days >= AMOUN_DAY_FOR_MIN_DISCOUNT && days < AMOUN_DAY_FOR_MAX_DISCOUNT) {
+    return PRICE_FOR_RENT * days - MIN_DISCOUNT;
+  }
+
+  if (days >= AMOUN_DAY_FOR_MAX_DISCOUNT) {
+    return PRICE_FOR_RENT * days - MAX_DISCOUNT;
+  }
+
+  return PRICE_FOR_RENT * days;
```

Comment:
It's good that you've used constants to avoid magic numbers in the code. However, you can further improve the function by combining the two `if` conditions. Since the second condition (`days >= AMOUN_DAY_FOR_MAX_DISCOUNT`) includes the first one (`days >= AMOUN_DAY_FOR_MIN_DISCOUNT && days < AMOUN_DAY_FOR_MAX_DISCOUNT`), you can check for the maximum discount first and then for the minimum discount, eliminating the need for the range check in the first condition.